### DISCUSSION TITLE
Example for custom fields

### DISCUSSION THREAD

**Discussion by bmamouri** [payloadcms/payload#8225] [Feature Requests & Ideas]
I am trying to create a new custom field using [Customizing Fields](https://payloadcms.com/docs/beta/admin/fields), but for the life of me I cannot figure out how it woeks. Basically I want to create a complex matrix component. The component itself is the easy part, but intergrating it with Payload CMS is the difficult part.

Is there any full example available for custom fields?

**Comment by tsemachh** [comment]
Check in discord:
https://discord.com/channels/967097582721572934/1146821621873643612/1267044140869488641
